region/norfair/west/Business Center.json

```diff
@@ -1972,6 +2319,29 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [4, 3],
+      "name": "Come in Shinecharging, Leave With Temporary Blue",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 8,
+          "openEnd": 0,
+          "gentleUpTiles": 4
+        }
+      },
+      "requires": [
+        "canLongChainTemporaryBlue",
+        {"or": [
+          "HiJump",
```

Comment:
Does this have a shorter runway, with the overhang?

Reply:
Oh right, yeah that needs to be split into a separate strat since the usable runway is 1 tile less in that case. Done.